lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -244,6 +247,17 @@
             // Install event handlers
             $( "#webcam_reset_button", this.el ).on( 'click', _.bind( this.reset, this ) );
             $( "#webcam_capture_button", this.el ).on( 'click', _.bind( this.capture, this ) );
+            $("#webcam_capture_button").keyup(function(event){
+                if(event.keyCode == key.enter){
+                        $("#webcam_capture_button").click();
+                    }
+                });
+
+            $("#webcam_reset_button").keyup(function(event){
```

Comment:
Nitpick: consistency. The previous lines are using .on( 'click' syntax, so it would make sense to use .on( 'keyup', function(event)... syntax to be consistent.